gapic-generator/templates/default/lib/_binding_override.text.erb

```diff
@@ -1,6 +1,7 @@
 <%- assert_locals package -%>
 <% @requires = capture do %>
 require "gapic/config"
+require "gapic/rest"
```

Comment:
I think we can remove the explicit `require "gapic/config"` above, since `gapic/rest` now requires your new file which in turn requires `gapic/config`. It's customary to include requires that are needed for what is referenced directly in the file, and `Gapic::Config` is no longer referenced directly here.